### ISSUE TITLE
Error in startNew should print a useful message

### ISSUE THREAD

**Issue by anthonychu** [Azure/azure-functions-durable-js#142]
Currently it's printing `Error` without any details. Examples:

* https://github.com/Azure/azure-functions-durable-js/issues/90#issue-454307747
* https://github.com/Azure/azure-functions-durable-js/issues/90#issuecomment-545149464

If possible, inspect other locations where errors are thrown and ensure they print the error message to logs as well.

**Comment by cgillum** [comment]
@anthonychu Is this with the 1.3.2 version of the npm package?

**Comment by anthonychu** [comment]
@cgillum No. There are a few other places with a similar (but different) problem: https://github.com/Azure/azure-functions-durable-js/pull/129#issuecomment-572300429
